server/src/main/java/org/apache/iotdb/db/newsync/transport/server/TransportServiceImpl.java

```diff
@@ -170,31 +182,32 @@ public TransportStatus transportData(
       }
 
       if (type != Type.FILE) {
-
         buff.position(pos);
         int length = buff.capacity();
         byte[] byteArray = new byte[length];
         buff.get(byteArray);
-        try (InputStream inputStream = new ByteArrayInputStream(byteArray);
-            DataInputStream dataInputStream = new DataInputStream(inputStream)) {
-          PipeData pipeData = PipeData.deserialize(dataInputStream);
-          // Do with file
-          // BufferedPipeDataQueue.offer(pipeData);
+        try {
+          PipeData pipeData = PipeData.deserialize(byteArray);
+          if (type == Type.TSFILE) {
+            // Do with file
+            handleTsFilePipeData((TsFilePipeData) pipeData, fileDir);
+          }
+          PipeDataQueueFactory.getBufferedPipeDataQueue(getPipeLogDirPath(identityInfo))
+              .offer(pipeData);
         } catch (IOException | IllegalPathException e) {
           e.printStackTrace();
```

Comment:
It's my fault. May use the statement like `logger.error("Error XXX", e);` is appropriate. The statement like `e.printStackTrace();` may not print to logger.

Reply:
fixed